apps/src/aichat/views/modelCustomization/constants.ts

```diff
@@ -26,7 +27,7 @@ export const EMPTY_MODEL_CARD_INFO: ModelCardInfo = {
   exampleTopics: [],
 };
 
-export const EMPTY_AI_CUSTOMIZATIONS: LevelAiCustomizations = {
+export const EMPTY_AI_LEVEL_CUSTOMIZATIONS: LevelAiCustomizations = {
```

Comment:
What do you think about naming this `DEFAULT_AI_LEVEL_CUSTOMIZATIONS since it includes default values for visibility?

Reply:
This will be deleted in the following PR, but will rename the other constant